app/assets/javascripts/components/form/form_body.es6.jsx

```diff
@@ -9,7 +9,13 @@ class FormBody extends Component {
   }
 
   get styles() {
-    return {};
+    return {
+      container: Object.assign( 
+        {},
+        StyleConstants.cards.default,
+        {}
```

Comment:
Why are you passing in an empty object here? Is it because you're predicting that we'll add styles to it in the future? If so, add `TODO(Sonia): ...` above this `get styles()` block! Also, don't forget them trailing commas.